### PR TITLE
docs: add slider page

### PR DESCRIPTION
**Pull request by didoesdigital**
This PR adds documentation for a Slider component and examples of visuals.

Branch preview:

- https://dev.cultureamp.design/di/add-slider-page/components/slider/

Closes: https://github.com/cultureamp/kaizen-design-system/issues/248

### CODE REVIEW

File: site/docs/components/slider.mdx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+## To keep in mind
+
+Sliders are helpful for choosing subjective values.
+
+- Always show the current, selected value.
+- Provide a visible label describing what the slider changes, such as "Brightness".
+- Show the minimum value on the left and maximum value on the right, where these are reversed in Right-To-Left (RTL) languages.
+- We only use horizontal sliders. We're yet to need a vertical slider.
+- We only use sliders with no fill so far.
```
Comment (okeeffed) [inline site/docs/components/slider.mdx:28]: The old performance slider looks to have that purple fill — is that an exception or deprecation?

File: site/docs/components/slider.mdx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+### Examples
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D47508%253A35978&chrome=DOCUMENTATION" allowfullscreen></iframe>
+
+## To keep in mind
+
+Sliders are helpful for choosing subjective values.
+
+- Always show the current, selected value.
```
Comment (okeeffed) [inline site/docs/components/slider.mdx:24]: I’m not fully sure if the new sliders show the current value but it sounds like this is something we should do (unless you mean just the visual “snap” to a value)

File: site/docs/components/slider.mdx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+- Provide a visible label describing what the slider changes, such as "Brightness".
+- Show the minimum value on the left and maximum value on the right, where these are reversed in Right-To-Left (RTL) languages.
+- We only use horizontal sliders. We're yet to need a vertical slider.
+- We only use sliders with no fill so far.
+- We only use sliders with a single handle. We're unlikely to need a slider with more than one handle.
+
+### Style
+
+- There's rarely a need to apply labels to every tick mark. Consider only showing a label for min, max, and middle ticks, such as 0%, 100%, and 50%.
```
Comment (okeeffed) [inline site/docs/components/slider.mdx:33]: It looks like we use ticks on the new 1-1 in Performance -- I just did a quick look. I wonder if that is more feedback for that slider? I think Performance needs to up its standards to align with this doc.

Screen Shot 2020-08-05 at 4 39 50 pm

File: site/docs/components/slider.mdx (hunk 1 above)
Comment (didoesdigital) [reply site/docs/components/slider.mdx]: It's a good use case to call out! Made some tweaks

File: site/docs/components/slider.mdx (hunk 3 above)
Comment (didoesdigital) [reply site/docs/components/slider.mdx]: Ticks is fine, tick *labels* is the key detail here. Updated the docs to try make it clearer.

**Comment by okeeffed** [comment]
LGTM 🎉

**Pull request merged**